tenable_io/api/workbenches.py

```diff
@@ -1,11 +1,30 @@
 from json import loads
 
+from six.moves import xrange
 from tenable_io.api.base import BaseApi
 from tenable_io.api.models import AssetList, AssetInfo, VulnerabilityList, \
     VulnerabilityOutputList
 
 
-class WorkbenchApi(BaseApi):
+class WorkbenchesApi(BaseApi):
+
+    CHAPTER_DIFF = u'diff'
+    CHAPTER_EXEC_SUMMARY = u'exec_summary'
+    CHAPTER_VULN_BY_ASSET = u'vuln_by_asset'
+    CHAPTER_VULN_BY_PLUGIN = u'vuln_by_plugin'
+    CHAPTER_VULN_HOSTS_SUMMARY = u'vuln_hosts_summary'
+
+    FILTER_TYPE_AND = u'and'
+    FILTER_TYPE_OR = u'or'
+
+    FORMAT_CSV = u'csv'
+    FORMAT_HTML = u'html'
+    FORMAT_NESSUS = u'nessus'
+    FORMAT_PDF = u'pdf'
+
+    REPORT_VULNERABILITIES = u'vulnerabilities'
+
+    STATUS_EXPORT_READY = u'ready'
```

Comment:
I don't believe they're documented anywhere. I will file a ticket to add them to our documentation.